harvardcards/static/js/modules/deck-view.js

```diff
@@ -254,8 +254,8 @@ function initModule() {
 				button_text.reverse();
 			}
 
-			$reveal_content_el.removeClass(css_cls[0]).addClass(css_cls[1]);
-			$button_el.text(button_text[0]);
+            $('.reveal').text(button_text[0]);
+            $('.reveal_content').removeClass(css_cls[0]).addClass(css_cls[1]);
```

Comment:
I meant to comment that -- it's because they wouldn't work with `accesskey`. `accesskey` does invoke the `click` handler, but the `this` that gets passed along isn't "quite right" (I wasn't able to determine exactly how it was different) and everything is relative to the passed `this`. 

I agree though, I'll modify this so it's passed in.